### DISCUSSION TITLE
❗ The source does not run in the terminal ❌

### DISCUSSION THREAD

**Discussion by ababyprogrammer** [organicmaps/organicmaps#9279] [Q&A] [answered]
Hello! I am one of your users.

I downloaded your source for some days. I looked at the majority of your code and everything was good. But I had a problem to run.

I followed all the steps you mentioned to run and test, but every time I got this error from terminal:
`command not found`

can you guide me

thank you

**Comment by gpesquero** [comment]
Hi @ababyprogrammer!!

First, please confirm that you're following the instructions described in [docs/INSTALL.md](https://github.com/organicmaps/organicmaps/blob/master/docs/INSTALL.md).

If you're following these instructions, please let me know...

1. Which is your development platform? (Linux, Windows, macOS)
2. Which Organic Maps app do you want to build? (Desktop, Android, iOS)
3. Which commands have you run successfully?
4. Which command is giving you the `command not found` error?

**Comment by ababyprogrammer** [comment]
Hello! Thank you for your answer.

1. I in Windows.
2. I want Desktop one.
3. No one. does not run any command.
4. `gradle build`

thank you

**Comment by hb0nd** [comment, marked as answer]
please re-read https://github.com/organicmaps/organicmaps/blob/master/docs/INSTALL.md

it's also not easily possible to build OM on Windows, you'll need to use WSL.

**Comment by ababyprogrammer** [comment]
Thanks. I'll try that.
